Misc/NEWS.d/next/Library/2024-07-14-11-18-28.gh-issue-120930.Kuo4L0.rst

```diff
@@ -0,0 +1,2 @@
+Do not add additional space within encoded word when folding
+:class:`email.message.EmailMessage`.
```

Comment:
I would phrase this: "Fixed a bug introduced by gh-92081 that added an incorrect extra blank to encoded words occurring in wrapped headers."

Reply:
Great suggestion, thanks! Just applied it. Should the PR title be aligned as well?

Reply:
Sure, that would probably make the commit log clearer.

Reply:
Done, I rephrased it to be more concise and fit 80 chars. Feel free to rename if you feel otherwise!